src/shared/utils/flowSchemaNormalizer.ts

```diff
@@ -133,10 +135,30 @@ export const normalizeSchema = (
     data?.field
   )
 
-  const filteredTags = filterTags(dedupedTags, lowerCasedSearchTerm)
+  const dedupedMeasurements = dedupeArray(measurements)
+
```

Comment:
I'm so confused, how is it even possible to have duplicate measurements if the parser is assigning the measurement as a key in an object

Reply:
lol it's your code man! i just moved it up a line. lemme know what you want me to do with it

Reply:
Let’s keep it for safety and the assumption that past Ariel wasn’t as out of it as current Ariel

Reply:
this should be done in the flux code IMO.

Reply:
Unfortunately I don't think the query builder will work properly if we return unique values with the flux results, since the schema is currently mapping out associated results in order to only allow users to build a valid query. Put simply, if we were to return unique values from the flux query, we might run the risk of losing the context of the existing associations between a measurement, field, and tag